### PR TITLE
check trashbin after deletion in UI tests

### PR DESCRIPTION
**Pull request by individual-it**
## Description
when deleting files in the test the trashbin should be checked to see if it works correctly

## Related Issue
https://github.com/owncloud/QA/issues/437

## Motivation and Context
more tests leads to less bugs
The [Simplified Smoke Test](https://github.com/owncloud/QA/blob/master/Server/Test_Plan_Simplified_Smoke_Test.md) requires to check the trashbin

## How Has This Been Tested?
run tests locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/ui/features/lib/FilesPage.php [hunk 1]
```diff
@@ -451,25 +154,24 @@ public function getTooltipOfFile($fileName, Session $session) {
...
 		) === false
 		) {
-			throw new UnexpectedPageException(
-				sprintf(
-					'Expected to be on "%s" but found "%s" instead',
-					$this->getUrl($urlParameters),
-					$this->getDriver()->getCurrentUrl()
-				)
-			);
+				throw new UnexpectedPageException(
```
Comment (phil-davis) [inline tests/ui/features/lib/FilesPage.php]: Somehow this block got a bonus indent. 
But that can easily be corrected in the next PR.

File: tests/ui/features/lib/FilesPageBasic.php [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+
+	/**
+	 * opens a file or navigates into a folder
+	 * 
+	 * @param string $name
+	 * @param Session $session
+	 * @return void
+	 */
+	public function openFile($name, Session $session) {
+		$fileRow = $this->findFileRowByNaBasicme($name, $session);
```
Comment (phil-davis) [inline tests/ui/features/lib/FilesPageBasic.php]: This does not look so good ``findFileRowByNaBasicme`` ?

**Review by phil-davis — CHANGES_REQUESTED** [review]
See comments.

File: tests/ui/features/lib/FilesPageBasic.php (hunk 2 above)
Comment (individual-it) [reply tests/ui/features/lib/FilesPageBasic.php]: CI failed because of that, fixed now

File: tests/ui/features/lib/FilesPage.php (hunk 1 above)
Comment (individual-it) [reply tests/ui/features/lib/FilesPage.php]: wondering why the pre commit hook did not shout at me

**Review by phil-davis — APPROVED** [review]
Looks good now. CI will find anything else :)

**Comment by phil-davis** [comment]
Note: files_texteditor UI tests pass on top of this. So ``FilesPage`` is correctly presenting a compatible interface, which is nice. No need to update UI tests in apps.

**Comment by individual-it** [comment]
backport to stable10 in #28995

**Pull request merged**